types/stripe/index.d.ts

```diff
@@ -1,11 +1,12 @@
-// Type definitions for stripe 4.9
+// Type definitions for stripe 5.0
```

Comment:
since this is a major version upgrade, please take a copy of the old definition files, and put it under a new folder `v4`. see https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version